x-pack/plugins/apm/typings/es_schemas/ui/fields/agent.ts

```diff
@@ -19,6 +19,7 @@ export type AgentName =
   | 'ruby';
 
 export interface Agent {
+  ephemeral_id?: string;
```

Comment:
Until now we've only added fields we were actively using. The other approach (of adding every single field) seemed like a waste of time since it is a manual process and most of the fields in the schema and never used by the ui.

Reply:
I'm actively using it in the Storybook. I'm ok with manually adding them, but it should be a reasonable assumption that I can copy real data from the UI, paste it into a story or test, and have it pass the type checker.